conf/evolutions/default/7.sql

```diff
@@ -0,0 +1,28 @@
+#
```

Comment:
Can you rename this file to 6.sql? The (previously) existing 6.sql has been removed from the master. Currently the latest sql file in master is 5.sql

Reply:
Done